### PR TITLE
fix: Use shebang `#!/usr/bin/env bash`

### LINKED ISSUE #385: shell scripts in tests have the path #! /bin/bash hardwired
Perhaps sth like
#!/usr/bin/env bash
would be better (once the todo.sh itself uses that)

Thanks
Ruda

### PR DESCRIPTION
**Pull request by hyperupcall**
**Before submitting a pull request,** please make sure the following is done:

- [x] Fork [the repository](https://github.com/todotxt/todo.txt-cli) and create your branch from `master`.
- [x] If you've added code that should be tested, add tests!
- [x] Ensure the test suite passes.
- [x] Format your code with [ShellCheck](https://www.shellcheck.net/).
- [x] Include a human-readable description of what the pull request is trying to accomplish.
- [x] Steps for the reviewer(s) on how they can manually QA the changes.
- [x] Have a `fixes #XX` reference to the issue that this pull request fixes.

This updates the shebang to `#/usr/bin/env bash`. It makes it compatible with not only FreeBSD and friends, but also NixOS. 

**QA'ing changes**

```sh
make test
```

Fixes #385

### CODE REVIEW

**Review by karbassi — COMMENTED** [review]
A clean PR! If the tests all pass, then we can merge it.

@inkarkat Do you see any issues with this change?

**Comment by inkarkat** [comment]
@karbassi Welcome back; it's good to hear from you again.

I don't see any problems, but would suggest that we work on the backlog of PRs first. Especially my #405 has been waiting for your approval for a long time, and touches many files. If you attack the PRs latest first, that'll only cause merge conflicts.

**Comment by hyperupcall** [comment]
@inkarkat @karbassi :wave:  I have updated this (any my other PRs) against the latest merges to `HEAD`

**Review by inkarkat — APPROVED** [review]

**Pull request merged**
